ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -32,23 +55,19 @@ StatusDialog {
         }
         StatusBaseText {
             Layout.alignment: Qt.AlignHCenter
-            text: "Selected account index: %1".arg(formData.selectedAccountIndex)
-        }
-        StatusBaseText {
-            Layout.alignment: Qt.AlignHCenter
-            text: "Selected network: %1".arg(formData.selectedNetworkChainId)
+            text: "Selected network: %1".arg(swapInputParamsForm.selectedNetworkChainId)
```

Comment:
Aren't these strings user facing? If yes then we should use `qsTr` for transaction and show the intention, right?

Reply:
Yup